### PR TITLE
Add brand tailwind colors

### PR DESCRIPTION
**Pull request by kylebonnici**

### CODE REVIEW

File: src/utils/colors.ts [hunk 1]
```diff
@@ -24,15 +26,125 @@ export const colors = {
...
-    indigo: '#3f51b5',
-    amber: '#ffc107',
-    purple: '#9c27b0',
-    green: '#4caf50',
-    deepPurple: '#673ab7',
-    orange: '#ff9800',
-    lime: '#cddc39',
-    pink: '#e91e63',
-    blueSlateLighter: '#7c98d3',
-    greenLighter: '#b1cbb3',
```
Comment (KievDevel) [inline src/utils/colors.ts:37]: Not sure if removing default colors won't break anything.

Comment (KievDevel) [reply src/utils/colors.ts:37]: If not used that's ok

File: Changelog.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 75 - UNRELEASED
+
+### Added
+
+-   Tailwind colors shades 50-100 for 'red', 'indigo', 'amber', 'purple',
```
Comment (KievDevel) [inline Changelog.md:14]:
```suggestion
-   Material UI color shades 50-900 for 'red', 'indigo', 'amber', 'purple',
```

File: Changelog.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ This project does _not_ adhere to
...
+    'green', 'deepPurple', 'orange', 'lime', 'lightGreen', 'lightBlue', 'pink',
+    such as e.g. `tw-bg-red-300`
+-   Default Tailwind color for 'red', 'indigo', 'amber', 'purple', 'green',
```
Comment (KievDevel) [inline Changelog.md:17]:
```suggestion
-   Default Material UI color for 'red', 'indigo', 'amber', 'purple', 'green',
```

File: src/utils/colors.ts (hunk 1 above)
Comment (kylebonnici) [reply src/utils/colors.ts:37]: We can't have both scales and the ones I deleted

**Review by KievDevel — APPROVED** [review]

**Pull request merged**